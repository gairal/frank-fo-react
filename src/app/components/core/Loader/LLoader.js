import React from 'react';
import { LinearProgress } from '@material-ui/core';

export default () => <LinearProgress mode="query" />;
