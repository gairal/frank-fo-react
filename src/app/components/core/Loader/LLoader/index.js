import LLoader from './LLoader';

export default LLoader;
