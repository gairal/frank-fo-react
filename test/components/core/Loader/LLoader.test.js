import React from 'react';
import renderer from 'react-test-renderer';
import LLoader from '@/components/core/Loader/LLoader';

describe('components > core > LLoader', () => {
  it('Renders correctly', () => {
    const component = renderer.create(<LLoader />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  // it('Appears when request fetching', () => {
  //   let component = renderer.create(<Loader />);
  //   let tree = component.toJSON();
  //   expect(tree).toMatchSnapshot();

  //   component = renderer.create(<Loader isFetching />);
  //   tree = component.toJSON();
  //   expect(tree).toMatchSnapshot();
  // });
});
