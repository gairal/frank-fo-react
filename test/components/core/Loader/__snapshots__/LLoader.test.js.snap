// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components > core > LLoader Renders correctly 1`] = `
<div
  className="MuiLinearProgress-root-1 MuiLinearProgress-colorPrimary-2"
  mode="query"
  role="progressbar"
>
  <div
    className="MuiLinearProgress-bar-9 MuiLinearProgress-barColorPrimary-10 MuiLinearProgress-bar1Indeterminate-12"
    style={Object {}}
  />
  <div
    className="MuiLinearProgress-bar-9 MuiLinearProgress-barColorPrimary-10 MuiLinearProgress-bar2Indeterminate-15"
    style={Object {}}
  />
</div>
`;
